src/middlewares/validator.js

```diff
@@ -1,8 +1,11 @@
 import { food } from '../models/store';
+import userModel from '../models/UserModel';
 
 export default (request, response, next) => {
     let isValid = 1;
-    if (request.body.foodItems) {
+    if (request.method === 'POST' && (request.originalUrl === '/api/v1/orders/' || request.originalUrl === '/api/v1/orders')) {
+        const count = (request.body.foodItems) ? (Array.isArray(request.body.foodItems)) ? request.body.foodItems.length : 0 : 0;
+        if (!request.body.foodItems === undefined || !count < 1) {
```

Comment:
I think you only need to check the count here, as the check above should have handled that check. Sync with me if you have questions about this.